src/convertToObject.js

```diff
@@ -3,16 +3,27 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const correctStyles = {};
+  const stylesFormatting = sourceString
+    .split(';')
+    .map(element => element.split(':')
+      .map(char => char.trim()))
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have same indentation. This makes code much more readable.

Also I'm not sure that `char` is a descriptive name for this variable
```suggestion
    .map(element => element
      .split(':')
      .map(char => char.trim()))
```